src/components/Tabs/__stories__/Tabs.stories.tsx

```diff
@@ -43,13 +53,25 @@ const Template: Story<
                 icon: args.withIcon ? gearIcon : undefined,
                 counter: args.withCounter ? Math.floor(Math.random() * 5 + 1) : undefined,
                 label: args.withLabel ? {content: 'Normal', theme: 'normal'} : undefined,
+                hasOverflow: args.withOverflow,
+                extraProps: {
+                    style: {
+                        maxWidth: args.withOverflow ? '100px' : 'auto',
+                    },
+                },
             },
             {
                 id: 'active',
                 title: 'Active Tab',
                 icon: args.withIcon ? gearIcon : undefined,
                 counter: args.withCounter ? Math.floor(Math.random() * 5 + 1) : undefined,
                 label: args.withLabel ? {content: 'Warning', theme: 'warning'} : undefined,
+                hasOverflow: args.withOverflow,
+                extraProps: {
+                    style: {
```

Comment:
I suggest adding a comment that this is for compatibility with the implementation of AdjustableTabs

Reply:
In this case this is just for demonstration of `hasOverflow` prop